compiler/modules/front/codeGen.bal

```diff
@@ -777,9 +777,23 @@ function codeGenAssignStmt(CodeGenContext cx, bir:BasicBlock startBlock, Environ
     if lValue is s:VarRefExpr {
         return codeGenAssignToVar(cx, startBlock, env, lValue.varName, expr);
     }
-    else {
+    else if lValue is s:MemberAccessExpr {
         return codeGenAssignToMember(cx, startBlock, env, lValue, expr);
     }
+    else {
+        return codeGenDestructuringAssignStmt(cx, startBlock, env, stmt);
+    }
+}
+
+function codeGenDestructuringAssignStmt(CodeGenContext cx, bir:BasicBlock startBlock, Environment env, s:AssignStmt stmt) returns CodeGenError|StmtEffect {
+    var { lValue, expr } = stmt;
+    if (lValue is s:WILDCARD) {
```

Comment:
style

But don't do this: call this function only if you have a wildcard

Reply:
Fixed